src/styles/nodeAttributes.ts

```diff
@@ -141,11 +139,11 @@ export class NodeAttributes extends ItemAttributes<INodeStyle> {
             }
           };
         } else if ('image' in style && typeof style.image === 'object') {
+          const urlType = Tools.getType(style.image.url as string);
           result = {
             image: {
               url:
-                Tools.getType(style.image.url as string) === 'imageUrl' ||
-                Tools.getType(style.image.url as string) === 'image'
+                urlType === 'imageUrl' || urlType === 'image'
```

Comment:
that doesn't fix the issue of passing a no image URL as Tools.getIn will still throw an error